testing/test_lucas.py

```diff
@@ -0,0 +1,34 @@
+"""Test module."""
+
+import lucas.lucas as p
```

Comment:
Why did you import as `p`?

If it's to save typing, you could have done something like `from lucas.lucas import MyLucas`. To call the namespace just `p` makes the meaning less clear in your code below.

Reply:
Yes it is to save typing.